content/00.build/40.tooling/20.hardhat/02.guides/30.migrating-to-zksync_old.md

```diff
@@ -0,0 +1,276 @@
+---
```

Comment:
You copied the Old migration file from https://github.com/matter-labs/zksync-docs/pull/163, but where is the new Migration page from that PR?